components/org.wso2.micro.integrator.ndatasource.capp.deployer/src/main/java/org/wso2/micro/integrator/ndatasource/capp/deployer/DataSourceCappDeployer.java

```diff
@@ -178,17 +179,16 @@ private DataSourceMetaInfo readDataSourceFile(File file) throws DeploymentExcept
         }
         try {
             InputStream in = new FileInputStream(file);
-
-            Document doc = DataSourceUtils.convertToDocument(in);
```

Comment:
This is the only usage of the DataSourceUtils#convertToDocument method. Shall we remove that also from the DataSourceUtils class?

Reply:
Fixed